resources/keywords/provision_dbinstance.resource

```diff
@@ -61,8 +60,8 @@ DBSC Instance Available On DataBase Access Page
     Element Should Be Visible    ${elemInstanceNameUpd}
     Element Should Be Visible    ${elemInstProvNameUpd}
 
-DBSC Instance Deployed Successfully
-    User Navigates To Connect Crunchy Bridge Screen On Developers View
+DBSC Instance For ${databaseProvider} Deployed Successfully
```

Comment:
To avoid the parameterization at line 45 and 63, we can make the variable ${databaseProvider} as Test Variable after line 41 in resources/keywords/deploy_instance_dev.resource, this way the validation keywords will be generic and won't require any arguments in their name

Reply:
Agree, But this will add more clarity on BDD perspective and the user can understand targeting DB provider. Is that fine?